riak_test/yz_solrq_test.erl

```diff
@@ -312,33 +310,22 @@ confirm_no_contenttype_data(Cluster, PBConn, BKey, Index) ->
 
 confirm_purge_strategy(Cluster, PBConn) ->
     confirm_purge_one_strategy(Cluster, PBConn,
-                               {?BUCKET5, ?INDEX5},
-                               {?BUCKET6, ?INDEX6}),
+                               {?BUCKET5, ?INDEX5}),
     confirm_purge_idx_strategy(Cluster, PBConn,
-                               {?BUCKET7, ?INDEX7},
-                               {?BUCKET8, ?INDEX8}),
-    confirm_purge_all_strategy(Cluster, PBConn,
-                               {?BUCKET9, ?INDEX9},
-                               {?BUCKET10, ?INDEX10}),
+                               {?BUCKET7, ?INDEX7}),
     confirm_purge_none_strategy(Cluster, PBConn,
-                                {?BUCKET11, ?INDEX11},
-                                {?BUCKET12, ?INDEX12}),
+                                {?BUCKET11, ?INDEX11}),
     ok.
 
-confirm_purge_one_strategy(Cluster, PBConn, Bucket1Index1, Bucket2Index2) ->
-    PurgeResults = do_purge(Cluster, PBConn, Bucket1Index1, Bucket2Index2,
+confirm_purge_one_strategy(Cluster, PBConn, Bucket1Index1) ->
+    PurgeResults = do_purge(Cluster, PBConn, Bucket1Index1,
```

Comment:
Minor formatting nitpick, but with the extra argument deleted, this doesn't need to be split across two lines anymore.